compiler/tests/utils.rs

```diff
@@ -316,6 +317,19 @@ pub fn bytes_token(s: &str) -> ethabi::Token {
     ethabi::Token::FixedBytes(ethabi::FixedBytes::from(s))
 }
 
+#[allow(dead_code)]
+pub fn bytes32(s: &str) -> Vec<u8> {
+    H256::from_str(s)
```

Comment:
Just pointing out that we try to not use single letter variable names such as `s`. That said, this file violates that rule numerous times and we don't need to fix that in this PR :)